### ISSUE TITLE
Error reading "mobile" RSK file

### ISSUE THREAD

**Issue by richardsc** [dankelley/oce#1482]
Have a data file that was collected with a new instrument, using an up-to-date version of the Ruskin mobile app. Trying to read it gives the error:
```r
> d <- read.oce('200268_20190118_2158.rsk')
Error in rsqlite_send_query(conn@ptr, statement) : 
  no such table: appSettings
```

I'm sure this is probably just an RSK sqlite schema change for the "EasyParse" (i.e. mobile format ... where is the schema document ????), but it will have become yet another case to check for when reading RSK files to see if it's one of the "normal" RSKs or one of the "mobile" ones (which is mostly just a stripped-down version, but it is also apparently missing tables).

**Comment by richardsc** [comment]
The problem is from this section of `read.rsk()`, which tries to get the Ruskin Version number:
```r
        ## Ruskin software version number
        appSettings <- RSQLite::dbReadTable(con, "appSettings")
        rv <- appSettings[1, 2]
        ##OLD rv <- read.table(pipe(cmd), sep="|")[1, 2]
        ruskinVersion <- as.numeric(strsplit(gsub(".[a-z].*$", "", gsub("^.*- *", "", rv)), "\\.")[[1]])
```

So, instead, it needs to do something like check if the `appSettings` table exists, and if not set `ruskinVersion` to something to indicate that it is a "mobile" file (easyParse).

**Comment by richardsc** [comment]
Fixed in commit aa4eb88c
